packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -142,6 +143,16 @@ function extractManifestOutput(output: string): string {
   return stripAnsi(output.substring(start, end + 1));
 }
 
+export function populateMetadata(angularVersion?: string): Metadata {
+  const packageJson = JSON.parse(readFileSync(`${__dirname}/../package.json`, "utf-8"));
+  return {
+    adapterNpmPackageName: packageJson.name.replace(/([@])/g, "\\$1"), // escape @ for yamlStringify
```

Comment:
can we include an example comment here, eg:

"myinitialstring" -> "myfinalstring"

Reply:
Added